CxxShim cannot be imported on Windows

```
S:\SourceCache\firebase\SwiftFirebase\Sources\FirebaseCore\FirebaseAuth+Swift.swift:5:8: error: no such module 'CxxShim'
import CxxShim
       ^
```

This is with the modulemap and headers being present in the SDK directory.

--- a/lib/ClangImporter/ClangIncludePaths.cpp
+++ b/lib/ClangImporter/ClangIncludePaths.cpp
@@ -38,7 +38,7 @@ static llvm::Optional<Path> getActualModuleMapPath(
   if (!SDKPath.empty()) {
     result.append(SDKPath.begin(), SDKPath.end());
     llvm::sys::path::append(result, "usr", "lib", "swift");
-    llvm::sys::path::append(result, platform, arch);
+    llvm::sys::path::append(result, platform);
     if (isArchSpecific) {
       llvm::sys::path::append(result, arch);
     }

